user-service/README.md

```diff
@@ -27,6 +27,13 @@ host operating system account or PKI based user authentication.
 The User Service separates this concern from the rest of the system. Other components use this service's API to request user details. 
 Some deployments may also allow Palisade to add users to the system, hence the presence of the `addUser()` method in the `UserService` interface.
 
+## High Level Architecture
+
+<!--- 
+See user-service/doc/user-service.drawio for the source of this diagram
+--->
+![User Service diagram](doc/user-service.png)
+
 ## Message Model and Database Domain
 
 | UserRequest     | UserResponse     | AuditErrorMessage | AuditableUserResponse |
```

Comment:
I think you could leave out the AuditableUserResponse, it's just used as an internal 'pair' and doesn't belong in this level of documentation

Reply:
Ok, removed the AuditableUserResponse details.